4.classify-single-cell-phenotypes/4b.derive-classification-profiles/4b.derive-classification-profiles.ipynb

```diff
@@ -0,0 +1,607 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.    print("Deriving multi class model classification profiles")
</pre><p>I like the indicator line here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/cell-health-data/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>